symplyphysics/definitions/damped_harmonic_oscillator_equation.py

```diff
@@ -0,0 +1,139 @@
+from sympy import (
+    Derivative,
+    symbols,
+    Function as SymFunction,
+    dsolve,
+    sqrt,
+    exp,
+    cos,
+)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    angle_type,
+    dimensionless,
+)
+from symplyphysics.core.expr_comparisons import expr_equals
+
+# Description
+## Assuming there is a damping force acting on an oscillating body that is linearly proportional
+## to the body's velocity, we can write a differential equation for the body's position. We're
+## assuming the body only moves in one direction.
+
+# Definition: d**2(x(t))/dt**2 + 2*zeta*omega*d(x(t))/dt + (omega**2)*x(t) = 0
+## x(t) - position of the oscillating body
+## t - time
+## omega - undamped angular frequency
+## zeta - damping ratio, the value of which critically determines the behavior of the system
+
+# not specifying the dimensions because it can be any physical quantity
+displacement = symbols("displacement", cls=SymFunction, real=True)
```

Comment:
I think you can use `length` dimension here. It is suitable for most cases and when we'd need more general equation, we'll refactor it.

Reply:
I agree